lib/matplotlib/tests/test_ticker.py

```diff
@@ -536,12 +536,66 @@ def test_formatstrformatter():
 
 
 @pytest.mark.parametrize('xmax, decimals, symbol, x, display_range, expected',
-                         percentformatter_test_cases)
```

Comment:
Accidentally copied instead of cutting, then forgot because the tests passed. Will fix.